src/web/components/TeamMember/TeamMembersTable.tsx

```diff
@@ -28,7 +30,10 @@ function TeamMembersTableContent({
   onRemoveTeamMember,
   onUpdateTeamMember,
 }: TeamMembersTableProps) {
+  const { LoggedInUser } = useContext(CurrentUserContext);
+
   const [showTeamMemberDialog, setShowTeamMemberDialog] = useState<boolean>(false);
+  const [allowTeamMemberActions, setAllowTeamMemberActions] = useState<boolean>(false);
```

Comment:
nit: perhaps `[showTeamMemberActions, setShowTeamMemberActions]` makes more sense here? Consistent with the other use state variable, and refers more to rendering than an actual permission check.

Reply:
Done!